src/homekit.cs

```diff
@@ -1671,12 +1683,65 @@ interface HMNumberRange {
 		NSNumber Max { get; }
 	}
 
+	[iOS (13,0), NoWatch, NoMac, NoTV]
+	[DisableDefaultCtor]
+	interface HMAccessoryOwnershipToken
+	{
+		[Export("initWithData:")]
+		IntPtr Constructor(NSData data);
+	}
+
+	[iOS (13,0), NoWatch, NoMac, NoTV]
+	[DisableDefaultCtor]
+	interface HMAddAccessoryRequest
+	{
+		[Export ("home")] 
+		HMHome Home { get; [NotImplemented] set; }
+		
+		[Export ("accessoryCategory")]
+		HMAccessoryCategory AccessoryCategory { get; [NotImplemented] set; }
+		
+		[Export ("accessoryName")]
+		NSString AccessoryName { get; [NotImplemented] set; }
+		
+		[Export ("requiresSetupPayloadURL")]
+		bool RequiresSetupPayloadURL { get; [NotImplemented] set; }
+		
+		[Export ("requiresOwnershipToken")]
+		bool RequiresOwnershipToken { get; [NotImplemented] set; }
+
+		[NullAllowed, Export ("payloadWithOwnershipToken:")]
+		HMAccessorySetupPayload GetPayloadWithOwnershipToken (HMAccessoryOwnershipToken ownershipToken);
+		
+		[NullAllowed, Export ("payloadWithURL:ownershipToken:")]
+		HMAccessorySetupPayload GetPayloadWithUrlAndOwnershipToken (NSUrl setupPayloadUrl, HMAccessoryOwnershipToken ownershipToken);
+	}
+
+	[iOS (13,0), Watch (6,0), TV (13,0), NoMac]
+	[DisableDefaultCtor]
+	interface HMNetworkConfigurationProfile : HMAccessoryProfile {
+		[Export ("delegate"), ArgumentSemantic.Weak)]
+		[NullAllowed]
+		HMNetworkConfigurationProfileDelegate WeakDelegate { get; set; }
+		
+		[Export ("isNetworkAccessRestricted")]
+		bool NetworkAccessRestricted { get; [NotImplemented] set; }
+	}
+
+	[iOS (13,0), Watch (6,0), TV (13,0), NoMac]
+	interface HMNetworkConfigurationProfileDelegate {
```

Comment:
It's probably incomplete, I didn't find 
```(void)profileDidUpdateNetworkAccessMode:(HMNetworkConfigurationProfile *)profile;```
in the todo file but it's here https://github.com/xamarin/xamarin-macios/wiki/HomeKit-iOS-xcode11-beta3

Reply:
Yup this looks like you want 

```csharp

interface IHMNetworkConfigurationProfileDelegate { }

[Watch (6,0), TV (13,0), NoMac, iOS (13,0)]
[Protocol, Model (AutoGeneratedName = true)]
[BaseType (typeof (NSObject))]
interface HMNetworkConfigurationProfileDelegate {

   [Export ("profileDidUpdateNetworkAccessMode:")]
   void DidUpdateNetworkAccessMode (HMNetworkConfigurationProfile profile);
}
```